thirdparty/NGT/lib/NGT/MmapManagerImpl.hpp

```diff
@@ -20,7 +20,11 @@
 #include "MmapManagerDefs.h"
 #include "MmapManagerException.h"
 
+#ifndef __MINGW64__
 #include <sys/mman.h>
+#else
+#include "../../annoy/src/mman.h"
```

Comment:
not reasonable to let NGT include annoy header file

Reply:
If you have no better way to resolve this issue, you can skip NSG and SPTAG.
These 2 indices are banned now since few customers are using them.